spec2nii/fileiobase.py

```diff
@@ -688,12 +688,12 @@ def __getitem__(self, key):
 
                 # check for out of range indexes
                 if v >= self.shape[i]:
-                    raise IndexError("index({}) out of range(0 <= index < {}) \
-                    in dimension {}".format(v, self.shape[i] - 1, i))
+                    raise IndexError(f"index({v}) out of range(0 <= index < "
+                                     f"{self.shape[i] - 1}) in dimension {i}")
 
                 if v <= (-1 * self.shape[i] - 1):
-                    raise IndexError("index({}) out of range(0 <= index < {}) \
-                    in dimension {}".format(v, self.shape[i] - 1, i))
+                    raise IndexError(f"index({v}) out of range(0 <= index < "
+                                     f"{self.shape[i] - 1}) in dimension {i}")
```

Comment:
By the way, I see an inconsistency between the actual test and the error message. Instead it should be something like this, shouldn't it?
```suggestion
                    raise IndexError(f"index({v}) out of range (0 >= index >= "
                                     f"{-1 * self.shape[i] - 1}) in dimension {i}")
```